examples/crd_derive_schema.rs

```diff
@@ -90,13 +90,13 @@ pub struct FooSpec {
 
     // Field with CEL validation
     #[serde(default = "default_legal")]
-    #[cel_validate(
+    #[x_kube(
         rule = Rule::new("self != 'illegal'").message(Message::Expression("'string cannot be illegal'".into())).reason(Reason::FieldValueForbidden),
         rule = Rule::new("self != 'not legal'").reason(Reason::FieldValueInvalid),
     )]
     cel_validated: Option<String>,
 
-    #[cel_validate(rule = Rule::new("self == oldSelf").message("is immutable"))]
+    #[x_kube(rule = Rule::new("self == oldSelf").message("is immutable"))]
```

Comment:
If we are renaming `cel_validate` to place it under `x_kube` we should probably make `validation` the key so it's clear that this is a reference to `x-kubernetes-validations` (the rule is already kind of clear by using the `Rule` struct);

```rust
#[x_kube(validation = Rule::new("self == oldSelf").message("is immutable"))]
```